### ISSUE TITLE
Windows 8.1 mouse coordinates, dispay DPI  issue

### ISSUE THREAD

**Issue by radupopa2010** [asweigart/pyautogui#31]
Today I was trying your module namely  "pyautogui.locateOnScreen('Summary.png')"

I took a screenshot of the button I wanted to click, but my program was miss clicking by some pixels .

the fix to this issue  in Windows 8.1 -> Dispaly -> Change the size of all times -> Small

by default in  windows 8.1  Change the size of all times  is Medium

**Comment by danforthf** [comment]
This is still the case with Windows 10. The screen size reported is the scaled value. My 2880x1620 display reports as 1920x1080 via the size() function when scaled to 175%. However the locateOnScreen function will only look for the image within the actual range of 1920x1080 pixels (starting top left). Setting scaling to 100% fixes this.

**Comment by Starbadger** [comment]
There's actually a better solution than setting your own scaling back to 1x. You can make a process "DPI aware" in windows with the following code:

```
from ctypes import windll
user32 = windll.user32
user32.SetProcessDPIAware()
```

**Comment by asweigart** [comment]
I've added the DPI scaling fix to 0.9.35, so hopefully this is now fixed.
